plugins/modules/znode.py

```diff
@@ -208,7 +216,7 @@ def check_params(params):
 class KazooCommandProxy():
     def __init__(self, module):
         self.module = module
-        self.zk = KazooClient(module.params['hosts'])
+        self.zk = KazooClient(module.params['hosts'], use_ssl=module.params['use_tls'])
```

Comment:
Do all versions of kazoo >= 2.1 support this?

Reply:
As far as I know, yes.